tests/mealpy_test.py

```diff
@@ -0,0 +1,116 @@
+from unittest import mock
+
+import pytest
+import requests
+import responses
+
+from mealpy import mealpy
+
+
+@pytest.fixture(autouse=True)
+def mock_responses():
+    with responses.RequestsMock() as _responses:
+        yield _responses
+
+
+def test_get_city(mock_responses):
+    response = {
+        'result': [
+            {
+                'id': 'mock_id1',
+                'objectId': 'mock_objectId1',
+                'state': 'CA',
+                'name': 'San Francisco',
+                'city_code': 'SFO',
+                'latitude': 'mock_latitude',
+                'longitude': 'mock_longitude',
+                'timezone': -7,
+                'countryCode': 'usa',
+                'countryCodeAlphaTwo': 'us',
+                'defaultLocale': 'en-US',
+                'dinner': False,
+                'neighborhoods': [
+                    {
+                        'id': 'mock_fidi_id',
+                        'name': 'Financial District',
+                    },
+                    {
+                        'id': 'mock_soma_id',
+                        'name': 'SoMa',
+                    },
+                ],
+            },
+            {
+                'id': 'mock_id2',
+                'objectId': 'mock_objectId2',
+                'state': 'WA',
+                'name': 'Seattle',
+                'city_code': 'SEA',
+                'latitude': 'mock_latitude',
+                'longitude': 'mock_longitude',
+                'timezone': -7,
+                'countryCode': 'usa',
+                'countryCodeAlphaTwo': 'us',
+                'defaultLocale': 'en-US',
+                'dinner': False,
+                'neighborhoods': [
+                    {
+                        'id': 'mock_belltown_id',
+                        'name': 'Belltown',
+                    },
+                ],
+            },
+        ],
+    }
+
+    mock_responses.add(
+        responses.RequestsMock.POST,
+        mealpy.CITIES_URL,
+        json=response,
+    )
+
+    mealpal = mealpy.MealPal(mock.sentinel.user)
+    city = mealpal.get_city('San Francisco')
+
+    assert city.items() >= {
```

Comment:
What does the `>=` do here? I have never used this way of asserting responses so it is quite unclear to me. Perhaps you can make use of the above mock response to do an equality check?

Reply:
This is a new pattern I'm trying out, inspired from jest javascript testing patterns. It's asserting that `city.items` is a superset of the minimum object we need, which contains values for `id` and `name`.

[items()](https://docs.python.org/3/library/stdtypes.html#dict-views) returns a set-like object, `>=` is [`issuperset`](https://docs.python.org/3/library/stdtypes.html#frozenset.issuperset). Otherwise the expected value would be a much bigger and more specified object. This replaces the deprecated [`unittest.TestCase.assertDictContainsSubset`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertDictContainsSubset) function.

Reply:
Interesting, the symbol makes more sense now.